aries-site/src/data/structures/components.js

```diff
@@ -1191,6 +1192,30 @@ export const components = [
       'spooling',
     ],
   },
+  {
+    name: 'Skeleton Loading',
```

Comment:
```suggestion
    name: 'Skeleton',
```
Should this be called "Skeleton" to align with the component name?

Reply:
good call